essential_reading/DataSources/Data_APIs.md

```diff
@@ -0,0 +1 @@
+# Data/APIs
```

Comment:
These files are all empty right now. These also feel like the types of topics that we don't want to write up documentation for because it probably already exists. I'm wondering if we can delete these and just add them to [00_StartHere.md](https://github.com/hytest-org/hytest/pull/60/files#diff-5b3eade2764901ac32b3593bf3ff555b8874abb2345fec7e01489b8fc11a3134)? What do you think?
![image](https://user-images.githubusercontent.com/30877272/190032850-d194eb33-3488-4b6a-964f-a21d747ad152.png)

Reply:
I stubbed them as placeholders. . . 

We could point to generic 'intake' (for example) documentation... my thought was to make something more on the lines of a demo (as opposed to tutorial) that would show the code sequences with good comments.  Using examples that look like the data they will be working with (catalog names, variable names, etc).  If done this way, we can breeze through future references to intake in the other notebooks. 

I guess what I'm saying is that starthere.md is a list of pointers to other places.  These other docs are canonical examples of how to use these concepts in an environment that looks just like what they will encounter as they read the rest of the material.